UPGRADE-4.2.md

```diff
@@ -5,3 +5,15 @@ Security
 --------
 
  * Using the `has_role()` function in security expressions is deprecated, use the `is_granted()` function instead.
+ * Passing custom class names for to the 
+   `Symfony\Component\Security\Core\Authentication\AuthenticationTrustResolver` to define
+   custom anonymouse and remember me token classes, is deprecated as of 4.2 and will be
+   removed in 5.0. To use custom tokens extend the existing AnonymousToken and RememberMeToken.
```

Comment:
The "and will be removed in 5.0" should be removed (same in the other paragraph). We decided to remove them all in deprecation as this should be "obvious".